tests/functional/SeeParsedBodyIsNotNullCept.php

```diff
@@ -0,0 +1,7 @@
+<?php 
```

Comment:
By using existing `POST /rest`, we could adjust the acception test to something like:

```php
<?php
$I = new FunctionalTester($scenario);

$data = ['testing' => '123'];
$I->wantTo('test that the parsed request body is returned in tact.');
$I->haveHttpHeader('content-type', 'application/json');
$I->sendPOST('/rest', $data);
$I->seeResponseCodeIs(\Codeception\Util\HttpCode::OK);
$I->seeResponseContainsJson([
    'rawBody' => json_encode($data),
    'formParams' => $data
]);
```